libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, inject } from '@angular/core';
+import { BrnTooltipDirective } from '@spartan-ng/ui-tooltip-brain';
+
+@Directive({
+	selector: '[hlmTooltip]',
+	standalone: true,
+	hostDirectives: [
+		{
+			directive: BrnTooltipDirective,
+			inputs: [
+				'brnTooltip: hlmTooltip',
+				'aria-describedby',
+				'disabled',
```

Comment:
Does this mean, that we are not able to disable the tooltip without disabling the host? 
I saw that you changed the input to only `disabled `in the brn in comparison to `matTooltipDisabled`.
How will we handle this, if we do not provide a seperate input for disabling only the tooltip?

Reply:
good point. I'll change this in a follow up PR!